app/decorators/flight_decorator.rb

```diff
@@ -0,0 +1,35 @@
+class FlightDecorator
+  attr_reader :component
+
+  def initialize(component)
+    @component = component
+  end
+
+  def departure_d
+    d_date = DateTime.parse(component.departure_datetime.to_s)
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.